evaluator/evaluator/app/routers/router_utils/router_utils.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import os
+from typing import Dict
+
+import requests
+from fastapi.exceptions import HTTPException
+
+logger = logging.getLogger(__name__)
+SKILL_MANAGER_API_URL = os.getenv(
+    "SKILL_MANAGER_API_URL", "https://square.ukp-lab.de/api/skill-manager"
+)
+
+
+def validate_access(token_payload: Dict):
```

Comment:
Wäre die Funktion hier nicht eher etwas für `evaluator/evaluator/auth_utils.py`?

Reply:
Ja, ich stelle die Funktion dann in ```evaluator/evaluator/auth_utils.py``` rein. Danke für den Hinweis!